docker.go

```diff
@@ -122,8 +124,12 @@ func (p Plugin) Exec() error {
 		}
 	}
 
-	cmds = append(cmds, commandRmi(p.Build.Name)) // docker rmi
-	cmds = append(cmds, commandPrune())           // docker system prune -f
+	if !p.Daemon.DisableRMI {
+		cmds = append(cmds, commandRmi(p.Build.Name)) // docker rmi
```

Comment:
I know it's not part of this PR, but what about using `docker image prune` instead of removing all?

Reply:
I think that definitely be neat, but would rather that be another discussion and subsequent PR.

Reply:
Yeah sure, just wanted to mention that because I have not seen this snippet before :)